src/main.ts

```diff
@@ -49,6 +49,18 @@ const main = async () => {
     }
 }
 
+!oneOff && process.on("uncaughtException", async function(e) {
+    if(action === "report") {
+        log(`Uncaught exception in ${argv.chainids.join(",")} ${action} process: ${e.toString()}. restarting in ${argv.rebootMinutes}`, 1)
+        setTimeout(syncRetryablesProcess, 1000 * 60 * argv.rebootMinutes);
+    } else if(action === "update" || action === "sync") {
```

Comment:
don't we need two separate conditions here? (i.e., `update` should run `updateProcess`)